_data/simple-icons.json

```diff
@@ -6957,6 +6957,16 @@
             "source": "https://medium.design/logos-and-brand-guidelines-f1a01a733592",
             "guidelines": "https://medium.design/logos-and-brand-guidelines-f1a01a733592"
         },
+        {
+            "title": "Medusa",
+            "hex": "8B5CF6",
```

Comment:
Given the website has had a bit of a refresh, do we think this hex is still relevant?

Reply:
Black might be more relevant now